src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/charts-and-dashboards-with-nrql.mdx

```diff
@@ -0,0 +1,58 @@
+redirects:
+---
+import dashboardsDashboardUi from 'images/dashboards_screenshot-crop_dashboard-ui.png'
+
+import chartsChartUi from 'images/queries-nrql_screenshot-crop_dashboard-chart-color.png'
+
+Knowing how to query your data is important for finding the information you need, but knowing how to visualize your data is important for making that information understandable for you and for your organization. Charts and dashboards are the key to data visualization, giving you multiple options that enable you to create visually appealing and easily understandable data streams. There are some key things to know about these features to see how they relate to your NRQL queries, and understanding them helps you make the most of your data.
+
+## Charts [#charts]
+
+<img
+  title="nrql-chart-example.png"
+  alt="A screenshot example of a chart"
+  src={chartsChartUi}
+/>
+<figcaption>
+  This is a chart created by a NRQL query showing the total number of transactions for each reported application across a period of 1 hour.
+</figcaption>
+
+Configuring a chart to represent your data is often the last step of creating a NRQL query. New Relic offers [multiple options](/docs/query-your-data/explore-query-data/use-charts/chart-types) to help you find the perfect way to visualize your data, and you can do many things with your charts once you've made them. You can:
+
+* Use your chart as the start of a [new dashboard](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards#dashboards-create) to monitor multiple data streams from a single location
+* [Add your chart to an existing dashboard](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/#add-pages-edit) to add another data stream to your monitoring
+* [Share your chart](/docs/query-your-data/explore-query-data/use-charts/use-your-charts#share-charts) with others by copying the permalink or downloading it as a PDF (or CSV, if applicable)
+* Edit the NRQL query behind the chart to help fine-tune the data you're seeing
+* [Create an alert](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition#create-simple-alert) from the chart to help proactively notify you when problems arise
+
+Charts are great on their own, but they really shine when you compile them together to create dashboards.
+
+## Dashboards [#dashboards]
+
+<img
+  title="nrql-dashboard-example.png"
+  alt="A screenshot example of a dashboard"
+  src={dashboardsDashboardUi}
+/>
+<figcaption>
+  A dashboard representing data in pie and bar chart formats.
+</figcaption>
+
+While charts are great for reporting on individual pieces, dashboards are the best way to get a much broader view of your data: if charts are pieces to a puzzle, dashboards are the puzzle completed. Dashboards are highly configurable, and you can use them to show you as many or as few data streams as you like. You can:
```

Comment:
I really like the puzzle metaphor!

I'm not sure you need the "as many or as few" when "as many" might suffice.

Reply:
Thanks! I tweaked the sentence slightly based on your feedback as well.